### DISCUSSION TITLE
Webtop + Crashplan It's possible?

### DISCUSSION THREAD

**Discussion by dcbasso** [jlesage/docker-crashplan-pro#429] [General]
Hi, I wish to use the crashplan together with webtop (https://docs.linuxserver.io/images/docker-webtop) project (I try debian and Ubuntu).
I give a lot of tries but does not works here. When I try to login I got an error, not clear error, and the service stops.

It's possible to make it work?

**Comment by jlesage** [comment]
What are you trying to do exactly ?  Running this container under Webtop ?

**Reply by dcbasso** [reply]
No, sorry as I see that you have a lot of experience wth Crashplan, I call you here to:
I wish to run CrashPlan inside the webtop/ubuntu or debian container, because the Crashplan is one tool that needs GUI to run, I have more two apps that needs GUI, that's why I'm trying to run webtop and crashplan inside.

**Reply by dcbasso** [reply]
When I try to run login in crashplan, the service stops. I do not know if it's related with:
- Ubuntu version (webtop uses Ubuntu 22, and crashplan run only at 18)
- Some fix that need to be done in the crashplan to run newer versions of ubuntu/debian
- Problem related to be related with the container itself

**Comment by jlesage** [comment]
I don't have experience with Webtop.  But since this is running a full desktop, CrashPlan should be installed like on a normal PC.  This might help: https://support.crashplan.com/hc/en-us/articles/8717685436173-Download-and-install-CrashPlan-for-Small-Business

Note that CrashPlan does support Ubuntu 22: see https://support.crashplan.com/hc/en-us/articles/8797351209485-Supported-operating-systems-Small-Business-

So if installing CrashPlan doesn't under Webtop, it might be related to the environment provided by Webtop.  Did you try to seek help on there forum ?

**Reply by dcbasso** [reply]
Well interessting, I think I look the wrong URL, since where I look just say about Ubuntu 18 (Old URL possibilly).
I will seek and to their support too.

**Comment by jlesage** [comment]
BTW, are you aware that this project (jlesage/docker-crashplan-pro) does provide access to the CrashPlan GUI via a web browser ?

**Reply by dcbasso** [reply]
Yes know and your project it's amazing BTW, but as I need to run at least 3 GUI apps, WEBTOP will do that for the 3 apps including Crashplan.

**Reply by jlesage** [reply]
Do you know if other apps you use have a container version ?  I find useful to have a container for each application I use.

**Reply by dcbasso** [reply]
No, they don't have any container.
I could be building and maintaining each one, but I this it's good enought to run over webtop at the moment. Huge container, but  works (not crashplan).

**Reply by jlesage** [reply]
Ok I see.

FYI, if you want to suggest new applications that would be nice to be dockerized, you can do it here: https://github.com/jlesage/docker-apps/issues

**Reply by dcbasso** [reply]
Possibly! I will take a look, maybe we can share some Knowledge, I know that I don't the same experience that you have, but... I have give a try and help

**Reply by dcbasso** [reply]
I test on VM and Crashplan works as needed, using Ubuntu 22.04. 
So it's something related with WEBTOP or the container. I trying to understand why this happens, logs doesn't help much
